src/t2iapi/device/device_responses.proto

```diff
@@ -15,6 +15,14 @@ import "t2iapi/biceps/metadata.proto";
 option java_package = "com.draeger.medical.t2iapi.device";
 option java_outer_classname = "DeviceResponses";
 
+/*
+Response containing all languages supported by the UI component represented by a MDS
+ */
+message GetMdsUiSupportedLanguagesResponse {
+    BasicResponse status = 1;
+    repeated string languages = 2;  // list of languages which are supported by the UI component represented by a MDS
```

Comment:
The `SetMdsUiLanguageRequest` restricts languages to RFC5646, does this also apply to these languages?

Reply:
they should, since those can be directly used for SetMdsUiLanguageRequest manipulation

Reply:
Then it should also be mentioned in this comment

Reply:
done